### ISSUE TITLE
Execution failed for task ':react-native-fetch-blob:processReleaseResources'

### ISSUE THREAD

**Issue by cirelramos** [RocketChat/Rocket.Chat.ReactNative#204]
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-fetch-blob:processReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

```

**Comment by diegolmello** [comment]
@cirelramos it seems Android SDK version missing.
Try running build with `--debug` to see more details.

**Comment by cirelramos** [comment]
[output](https://drive.google.com/open?id=1MN5il0sYB0dPSmy8b0OhU4pT3BvxM-sC)
[picture](https://drive.google.com/open?id=1NJrIKAJtZBSAlSWk68iNjyfD_bACguyh)
 

`./gradlew installDebug --debug > /tmp/output-send.txt              
16:52:32.781 [ERROR] [org.gradle.BuildExceptionReporter] 
16:52:32.783 [ERROR] [org.gradle.BuildExceptionReporter] FAILURE: Build failed with an exception.
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] 
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] * What went wrong:
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] Execution failed for task ':react-native-fetch-blob:processReleaseResources'.
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] > com.android.ide.common.process.ProcessException: Failed to execute aapt
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] 
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] * Try:
16:52:32.786 [ERROR] [org.gradle.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.`

**Comment by diegolmello** [comment]
```
Process 'command '/home/server/Android/Sdk/build-tools/23.0.1/aapt'' finished with exit value -1 (state: FAILED)
```

Install SDK version 25.0.1 and try again.

**Comment by cirelramos** [comment]
i dont understand, i removed the sdk 23.0.1, installed the 25.0.1 and run ./gradlew installDebug --debug, but start again download the sdk 23.0.1 :s wtf

**Comment by diegolmello** [comment]
It's an Android installation trouble and it seems and it seems to be solved on #208.
I'm closing.

**Comment by cirelramos** [comment]
i solved with this -> sudo apt-get install libncurses5:i386 libstdc++6:i386 lib32z1 libbz2-1.0:i386
